scAR-cpu.yml

```diff
@@ -0,0 +1,23 @@
+name: scAR
+channels:
+- nvidia
+- pytorch
+- conda-forge
+dependencies:
+- conda-forge::python>=3.8.6
```

Comment:
I am not sure if this is a good practice, but I specified from where to fetch the dependencies